notebooks/post-startup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Default post startup script for AI notebooks.
+# AI Notebook post startup scrips are run only when the instance is first created.
+
+# Send stdout and stderr from this script to a tmp file for debugging.
+exec >> /tmp/post-startup-output.txt
+exec 2>&1
+set -x
+
+# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# user space.
+JUPYTER_USER="jupyter"
+
+# Install common packages in conda environment
+/opt/conda/bin/conda install -y pre-commit nbdime nbstripout pylint pytest
+# Install nbstripout for the jupyter user in all git repositories.
+sudo -u ${JUPYTER_USER} sh -c "/opt/conda/bin/nbstripout --install --global"
+
+# Install & configure the Terra CLI
+cd /tmp || exit
```

Comment:
Should this line be moved up before the package installations, since the stdout/err file will not be generated if `/tmp` doesn't exist?